projects/plugins/boost/composer.json

```diff
@@ -30,12 +30,15 @@
 		"yoast/phpunit-polyfills": "0.2.0"
 	},
 	"scripts": {
-		"fix": [
+		"phpcs:fix": [
 			"composer --working-dir=../../../ phpcs:fix ./projects/plugins/boost"
 		],
-		"lint": [
+		"phpcs:lint": [
 			"composer --working-dir=../../../ phpcs:lint ./projects/plugins/boost"
 		],
+	  	"phpcs:compatibility": [
+			"composer --working-dir=../../../ phpcs:compatibility ./projects/plugins/boost"
+		],
```

Comment:
Is this needed at all, given that you can do it from the monorepo?

See the Backup plugin for example:
https://github.com/Automattic/jetpack/blob/6621fa998d01d7fd95ad7c9ed2915c260bf9676d/projects/plugins/backup/composer.json#L24-L33

Reply:
Yes, I was debating for these, I thought they can be used as aliases so a composer command could be run from the boost directory if someone wanted. I guess from the monorepo, someone would have to run `composer phpcs:fix ./projects/plugins/boost" which is a bit longer to type (though we have history and autocomplete in command line). Alternatively the `pnpm` commands could refer directly to the composer commands in the monorepo. 

Any thought @thingalon ?

Reply:
I think that we should encourage developers to use the monorepo "way of doing things". While it does mean we have a longer command to type initially, it makes it easier for devs used to Boost to dabble in other Monorepo projects in the future.

IMHO, if we want to make the commands we have to run shorter, we could set up something that lets devs in the Jetpack Monorepo "set" which project they're focused on via an environment variable, and allow the composer instructions to read from that env variable if a value isn't explicitly set as a shortcut.

Reply:
With https://github.com/Automattic/jetpack/pull/20669/commits/3acce156eba89b6c8f376e69ccb711705bb41c88 I have removed the additional command and updated Jetpack Boost documentation so it is more aligned with the Jetpack Monorepo processes.